Lib/test/test_exceptions.py

```diff
@@ -1089,7 +1089,7 @@ def test_attributes(self):
         self.assertEqual(exc.name, 'somename')
         self.assertEqual(exc.path, 'somepath')
 
-        msg = "'invalid' is an invalid keyword argument for this function"
+        msg = "'invalid' is an invalid keyword argument for ImportError()"
```

Comment:
`msg` is a regular expression. It is matched only by accident. Either escape or remove `()`.